crates/sloth_vm/src/sloth_std/misc.rs

```diff
@@ -0,0 +1,43 @@
+use super::NATIVE_LIBRARY;
+use crate::native::{self, NativeFunction, NativeFunctionResult};
+use crate::value::{Object, ObjectType, Primitive};
+use crate::VM;
+
+fn get_doc(vm: &mut VM, args: &[Primitive]) -> NativeFunctionResult {
+    let Some(Primitive::Object(ptr)) = args.get(0).cloned() else {
+        return Err(native::Error::InvalidArgument);
+    };
+
+    let object = vm
+        .objects()
+        .get(ptr as usize)
+        .ok_or(native::Error::InvalidArgument)?;
+
+    let ObjectType::String(str) = &object.typ else {
```

Comment:
This should take in a function as opposed to a string. Example usage could be as follows:
```
get_doc(file$write)
```

Functions are first class so passing in a function as a value shouldn't be an issue.

Reply:
```suggestion
    let ObjectType::NativeFunction(fnc)= &object.typ else {
        return Err(native::Error::InvalidArgument);
    };

    let docs = NATIVE_LIBRARY
        .get(fnc.name.clone())
```

This should do it right?